### DISCUSSION TITLE
Towards a better GUI

### DISCUSSION THREAD

**Discussion by thetwom** [thetwom/toc2#74] [Ideas]
Feel free to share proposals and ideas towards a better GUI ...

**Comment by RustoMCSpit** [comment]
https://opensourcedesign.net/

**Reply by RustoMCSpit** [reply]
It's a project to improve the design of open source apps. So it provides resources, connects designers (like OP) with engineers/projects, spreads awareness etc.

**Comment by RustoMCSpit** [comment]
It's a project to improve the design of open source apps. So it provides resources, connects designers (like OP) with engineers/projects, spreads awareness etc.

**Comment by RustoMCSpit** [comment]
> Feel free to share proposals and ideas towards a better GUI ...

https://opensourcedesign.net/jobs/ apply the project here!!

**Reply by thetwom** [reply]
Thanks for the suggestions. But I rather wanted to give here the possibility to share ideas instead of trying to actively trying to find collaborators. I cannot promise that I will indeed find time to implement a refactored UI, so reaching out for help is not appropriate here.
